tests/testScrapPaper.js

```diff
@@ -171,26 +172,26 @@ describe('Barcode generation, student submission, and scanning process', functio
           assert.equal(recomputed_checksum, checksum);
         });
       });
-      // it('barcodes should ALL be scannable by barcode reader', async () => {
-      //   decodedJpegs = await decodeBarcodes(
-      //     pdfBuffer,
-      //     `${config.imageProcessingDir}/test`,
-      //     'Any original filename.pdf',
-      //     console
-      //   );
-      //   assert.isDefined(decodedJpegs);
-      //   decodedJpegs.forEach((jpeg) => {
-      //     assert.isNotNull(jpeg.barcode);
-      //   });
-      // });
-      // it('pdf barcodes should checksum successfully against base36 barcode component', () => {
-      //   decodedJpegs.forEach((jpeg) => {
-      //     const { base36, checksum } = getBarcodeSegments(jpeg.barcode);
-      //     const recomputedChecksum = jsCrc.crc16(base36);
-      //     assert.equal(recomputedChecksum, checksum);
-      //     validBarcodes.push(jpeg.barcode);
-      //   });
-      // });
+      it('barcodes should ALL be scannable by barcode reader', async () => {
+        decodedJpegs = await decodeBarcodes(
+          pdfBuffer,
+          `${config.imageProcessingDir}/test`,
+          'Any original filename.pdf',
+          console
```

Comment:
Ideally this should be a no-op logger instead of dumping everything to the console.

Reply:
Can we get a no op logger without implementing more code?

Reply:
Done.